tools/SourceKit/CMakeLists.txt

```diff
@@ -84,6 +84,11 @@ function(add_sourcekit_default_compiler_flags target)
     ANALYZE_CODE_COVERAGE "${analyze_code_coverage}"
     RESULT_VAR_NAME link_flags)
 
+  if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
```

Comment:
Absolutely. I'm worried about [SR-710](https://bugs.swift.org/browse/SR-710) getting done in time for Swift 3.0, so I'm reducing scope a bit here. Ideally, I'd like SourceKit to run everywhere. Were it able to do so, we'd probably want to change this to handle more than just Linux.